cmd/commento/main.go

```diff
@@ -17,6 +17,10 @@ func main() {
 		Die(err)
 	}
 	
+	if err := godotenv.Load(); err != nil {
```

Comment:
For the record, this is what the original ruby library does:

> dotenv-rails will load the following files, starting from the bottom. The first value set (or those already defined in the environment) take precedence:
> .env - The Original®
> .env.development, .env.test, .env.production - Environment-specific settings.
> .env.local - Local overrides. This file is loaded for all environments except test.
> .env.development.local, .env.test.local, .env.production.local - Local overrides of environment-specific settings.

That not what the godotenv library does *yet* so I've simulated it be loading the files in that order. I'll see if I can get ahold of the developer of godotenv and have him add those defaults